src/Util/FileMatcherPattern.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PHPUnit\Util;
+
+class FileMatcherPattern
+{
+    public function __construct(public string $path)
```

Comment:
this will/could include the `$suffix`, `$prefix`  and `$exclude` also

Reply:
or could simply remove this and handle the prefix/suffix independently of the "file matcher"